docs/insert-version-url.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+"""
+Insert an entry into the version dropdown json file
+
+Usage: add_version_dropdown.py your.version.here
+"""
+
+import json
+import sys
+import os
+
+
+def create_url_entry(version):
+    url = f"https://docs.determined.ai/{version}/"
+
+    entry = {"version": version, "url": url}
+
+    return entry
+
+
+def insert_entry(json_file, entry, entry_position=1):
+    with open(json_file, "rb") as f:
+        all_urls = json.load(f)
+
+    all_urls.insert(entry_position, entry)
+
+    with open(json_file, "w") as f:
+        json.dump(all_urls, f, indent=4 * " ")
+        f.write("\n")
+
+
+def insert_version_url(json_file, version):
+    new_entry = create_url_entry(version)
+    insert_entry(json_file, new_entry)

```

Comment:
I don't know that we really need a whole other six-line function to create a dict with two entries.
```suggestion
    insert_entry(json_file, {"version": version, "url": f"https://docs.determined.ai/{version}/"})
```